src/main/java/main/controllers/Project/SuiteController.java

```diff
@@ -86,4 +90,60 @@ public boolean delete(TestSuiteDto template) throws AqualityException {
         }
         return testSuites;
     }
+
+    public void syncLegacyTests(List<TestDto> legacyTests, Integer suiteId, boolean removeNotExecutedResults) throws AqualityException {
+        if(baseUser.isManager() || baseUser.getProjectUserBySuiteId(suiteId).isManager()){
```

Comment:
Are you sure that sync should make only manager?
I think test results are responsibility of engineer.

Reply:
fixed